tests/test_dupefilters.py

```diff
@@ -120,14 +120,14 @@ def test_request_fingerprint(self):
 
         dupefilter.close('finished')
 
-        class CaseInsensitiveRFPDupeFilter(RFPDupeFilter):
+        class RequestFingerprinter:
```

Comment:
How hard would it be to check that from_crawler or from_settings methods are really called for RequestFingerprinters?